user_docs/en/userGuide.md

```diff
@@ -1753,7 +1753,7 @@ This option is a checkbox that, when checked, tells NVDA to automatically save t
 ##### Show exit options when exiting NVDA {#GeneralSettingsShowExitOptions}
 
 This option is a checkbox that allows you to choose whether or not a dialog appears when you exit NVDA that asks what action you want to perform.
-When checked, a dialog will appear when you attempt to exit NVDA asking whether you want to exit, restart, restart with add-ons disabled or install pending updates (if any).
+When checked, a dialog will appear when you attempt to exit NVDA, asking whether you want to exit, restart, restart with add-ons disabled and debug logging enabled, restart with debug logging enabled, or install pending updates (if any).
```

Comment:
this might be better served as a list now. This sentence is quite long

Reply:
Done